shared/crypto/input/index.stories.tsx

```diff
@@ -1,126 +1,109 @@
 import * as React from 'react'
 import * as Sb from '../../stories/storybook'
+import * as Container from '../../util/container'
 import * as Constants from '../../constants/crypto'
-import {TextInput, FileInput} from '.'
+import HiddenString from '../../util/hidden-string'
+import {Input} from '.'
 
-const onClearInput = Sb.action('onClearInput')
-const onChangeText = Sb.action('onChangeText')
-const onSetFile = Sb.action('onSetFile')
+const cryptoCommon = {
+  input: new HiddenString(''),
+  inputType: 'text',
+}
+const storeCommon = Sb.createStoreWithCommon()
+const store = {
```

Comment:
can use produce since you have it already

Reply:
Didn't even need the produce at all, the common store was enough since it hits the crypto reducer and intializes